### PR TITLE
Kifi tagging js

### PR DESCRIPTION
**Pull request by joonhocho**
@2is10 @andrewconner 
Would you like to review before pull?
I will also re-review all changes before pull.

### CODE REVIEW

File: kifi-backend/modules/shoebox/public/css/main.css [hunk 1]
```diff
@@ -67,29 +69,47 @@ body[data-view=profile] .my-pic::before { transition: none; animation-name: edit
...
 .tag-icon {background-image: url(../img/tag-icon-gray.png)}
 .collection.active .tag-icon {background-image: url(../img/tag-icon-white.png)}
+.collection.renaming .tag-icon {background-image: url(../img/tag-icon-gray.png)}
+.collection.tag-new {color: #858585; background-color: #d4e4f9}
+.collection.tag-new a {padding-top: 9px; padding-bottom: 9px}
+#collections .collection.tag-new {line-height: normal}
+.collection.tag-new:hover {background-color: #c4d4e9}
+.collection.tag-new.highlight {background-color: #6fa6ef}
+.tag-new .caption {display: block; font-size: 11px; font-weight: 500; text-transform: uppercase; margin-bottom: 1px}
+.tag-new .tag-create {float: left; position: relative; left: -4px; margin: 5px 4px 0 0; width: 18px; height: 18px; border-radius: 50%; background: url(../img/plus.png) center/8px no-repeat #b8c5d2; image-rendering: -moz-crisp-edges; image-rendering: -webkit-optimize-contrast; image-rendering: crisp-edges}
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/css/main.css:111]: Let’s use retina versions of all of our images. `image-rendering` is [not well supported](https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering#Browser_compatibility), and we’re targeting IE 9 with the website.

Comment (2is10) [inline kifi-backend/modules/shoebox/public/css/main.css:111]: Of course I mean both standard _and_ retina versions, to avoid browser scaling.

**Comment by 2is10** [comment]
Looks good.

If you don’t mind, please delete all of the functions from util.js that are not currently in use. Best to avoid including dead code that’s our own.

**Comment by joonhocho** [comment]
Thanks for reviewing. I will address your comments for the next release.
I getting pressure for release.

**Pull request merged**